src/components/forms/reportSiteForm/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Radio, Input, Typography, Form, FormInstance } from 'antd';
+import { SubmitButton } from '../../themedComponents';
+import { requiredRule } from '../../../utils/formRules';
+import { ReportSiteRequest } from '../../../containers/treePage/ducks/types';
+import { useTranslation } from 'react-i18next';
+import { site } from '../../../constants';
+import { n } from '../../../utils/stringFormat';
+
+const ItemLabel = styled(Typography.Paragraph)`
+  line-height: 0px;
+`;
+
+interface ReportSiteFormProps {
+  form: FormInstance;
+  onFinish: (reportInfo: ReportSiteRequest) => void;
+}
+
+const ReportSiteForm: React.FC<ReportSiteFormProps> = ({ form, onFinish }) => {
+  const { t } = useTranslation(n(site, ['forms']), {
+    nsMode: 'fallback',
+  });
+
+  const reasonOptions = [
+    {
+      value: 'Inappropriate Content',
+      label: t('report_site.options.inappropriate'),
+    },
+    {
+      value: 'Incorrect Information',
+      label: t('report_site.options.incorrect'),
+    },
```

Comment:
At least from the API spec it doesn't seem like reason is categorical but I can see why it makes sense to simplify it to these two cases. In case someone would want to add more reasons in the future maybe this could be a dropdown instead? This is just an idea but the radio is perfectly fine as is

Reply:
I like the dropdown idea - like you mentioned it'll definitely scale better with more reasons. I kept the API generic so if we do want to add more reasons, we only have to change the frontend, but we can still limit the users' options here so they don't make reports for random things.

Reply:
coolio!